src/persist/src/indexed/cache.rs

```diff
@@ -100,7 +100,11 @@ impl<B: BlobRead> BlobCache<B> {
     }
 
     /// Synchronously fetches the batch for the given key.
-    fn fetch_unsealed_batch_sync(&self, key: &str) -> Result<Arc<BlobUnsealedBatch>, Error> {
+    fn fetch_unsealed_batch_sync(
+        &self,
+        key: &str,
+        add_to_cache: bool,
```

Comment:
let's make this an enum instead of a bool (like the Atomicity one in Blob)

Reply:
done - struggled with the names feel free to bikeshed those pls!

Reply:
I like them! though I would s/Hint/CacheHint/ (even though it's in mod cache, I think the import stutter is fine)

Reply:
done